--- irqbalance-1.12/irqbalance/classify.c~	2006-07-29 00:15:24.000000000 -0400
+++ irqbalance-1.12/irqbalance/classify.c	2006-07-29 00:20:43.000000000 -0400
@@ -57,6 +57,9 @@ char *storage_modules[] = {
 	"libata",
 	"ohci1394",
 	"sym53c8xx",
+	"ata_piix",
+	"sata",
+	"ahci",
 	0
 };
 
@@ -64,16 +67,20 @@ char *ethernet_modules[] = {
 	"eth",
 	"e100",
 	"eepro100",
-	"orinico_cs",
+	"orinoco_cs",
 	"wvlan_cs",
-	"3c5",	
+	"3c5",
 	"HiSax",
+	"ipw2",
+	"via_rhine",
+	"sis900",
 	0
 };
 
 char *gige_modules[] = {
 	"e1000",
 	"tg3",
+	"r8169",
 	0
 };
 
