--- irqbalance/Makefile.orig	2003-10-24 10:51:51.000000000 +0200
+++ irqbalance/Makefile	2003-12-15 01:12:59.403007304 +0100
@@ -3,7 +3,7 @@
 	$(CC) $(CFLAGS) irqbalance.c cpuinfo.c procinterrupts.c classify.c policy.c  -o irqbalance
 
 debug: irqbalance.c cpuinfo.c prototypes.h Makefile procinterrupts.c classify.c policy.c  debug.c
-	$(CC) irqbalance.c cpuinfo.c procinterrupts.c classify.c policy.c debug.c -DDEBUG -o irqbalance
+	$(CC) $(CFLAGS) irqbalance.c cpuinfo.c procinterrupts.c classify.c policy.c debug.c -DDEBUG -o irqbalance
 
 irqbalance.1: irqbalance.xml
 	xmlto man irqbalance.xml
